docs/use_cases.md

```diff
@@ -0,0 +1,72 @@
+<!-- omit in toc -->
+# Use Cases
+
+**Navigation**
+- [Requirements](./requirements.md)
+- [README](../README.md)
+
+**Table of Contents**
+- [UC01 - Application Start](#uc01---application-start)
+  - [Description](#description)
+  - [Generated Requirements](#generated-requirements)
+  - [Implementation Details](#implementation-details)
```

Comment:
Should we add Main actor and Pre/Post-conditions to the use cases structure? Or should we wait for when we start writing them more in detail?

Reply:
Good catch. Maybe we could discuss this together during the next meeting?